akka-actor-tests/src/test/scala/akka/actor/ReceiveTimeoutSpec.scala

```diff
@@ -102,7 +102,12 @@ class ReceiveTimeoutSpec extends AkkaSpec {
         }
       }))
 
-      val ticks = system.scheduler.schedule(100.millis, 100.millis, timeoutActor, TransperentTick)(system.dispatcher)
+      val ticks = system.scheduler.schedule(100.millis, 100.millis, new Runnable {
+        override def run() = {
+          timeoutActor ! TransperentTick
+          timeoutActor ! Identify(None)
+        }
+      })(system.dispatcher)
```

Comment:
Then I think there should be a separate test case rather than appending it to an existing one like this.